README.md

```diff
@@ -1,12 +1,3 @@
 # zed-kotlin
-Syntax highlighting for Kotlin in [Zed](https://github.com/zed-industries/zed).
 
-### Test locally
```

Comment:
I don't think we should be removing that section from README.

Reply:
I may be misunderstanding how this works but this seems outdated?

- grammars dir is generated from repo specified in `extension.toml`
- zed now has nice ui for installing dev extension

I definitely could be missing something though

Reply:
Oh yeah, that's a fair point. Let's proceed with this change then.